src/core_modules/capture-core/components/Pages/StageEvent/StageEventList/StageEventList.component.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+import React from 'react';
+import type { ComponentType } from 'react';
+import withStyles from '@material-ui/core/styles/withStyles';
+import { compose } from 'redux';
+import type { Props } from '../StageEventListPage.types';
+
+const getStyles = () => ({});
+
+const StageEventListComponentPlain = () => (<>
+    <div>Stages and Events</div>
+</>);
+
+export const StageEventListComponent: ComponentType<$Diff<Props, CssClasses>> =
+  compose(
```

Comment:
Hey, 
I was wondering what is the need to use `compose` here when there is only one HOC? Looking to other components we don't use `compose` for this case. For example https://github.com/dhis2/capture-app/blob/master/src/core_modules/capture-core/components/WidgetProfile/WidgetProfile.component.js#L114

Thanks!

Reply:
Hey, I plan to add a HOC to handle error and loading later on. But okay to remove